# avoid installation into python module's SP dir.
# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/08
diff -ru eman2-2.99.47/cmake/Python.cmake eman2-2.99.47_Python3_SITELIB/cmake/Python.cmake
--- eman2-2.99.47/cmake/Python.cmake	2023-03-04 20:27:15.000000000 +0100
+++ eman2-2.99.47_Python3_SITELIB/cmake/Python.cmake	2023-08-23 17:24:24.825880499 +0200
@@ -1,7 +1,7 @@
 set(Python3_FIND_VIRTUALENV ONLY)
 
 find_package(Python3 REQUIRED  COMPONENTS Interpreter Development NumPy)
-
+set(Python3_SITELIB ${CMAKE_INSTALL_PREFIX}/lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages)
 cmake_print_variables(Python3_EXECUTABLE)
 cmake_print_variables(Python3_LIBRARIES)
 cmake_print_variables(Python3_INCLUDE_DIRS)
